go/libkb/keyfamily.go

```diff
@@ -315,40 +315,15 @@ func (cki ComputedKeyInfos) InsertServerEldestKey(eldestKey GenericKey, un Norma
 
 func (ckf ComputedKeyFamily) InsertEldestLink(tcl TypedChainLink, username NormalizedUsername) (err error) {
 	kid := tcl.GetKID()
-	key, err := ckf.FindKeyWithKIDUnsafe(kid)
+	_, err = ckf.FindKeyWithKIDUnsafe(kid)
 	if err != nil {
 		return
 	}
```

Comment:
I don't think it would ever return `nil`, unless there was no key created with the "eldest" link